documentation/release_notes.rst

```diff
@@ -8,6 +8,39 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2021.6.1
+===============
+
+New Features
+------------
+- Added support of unified shared device memory (USM) for ``reduce_by_segment`` algorithm.
+
+Fixed Issues
+------------
+- Fixed compilation errors happening when using C++20.
+- Fixed ``CL_OUT_OF_RESOURCES`` issue which used to occur during execution of Radix sort algorithm on CPU devices.
+- Fixed error in ``exclusive_scan_by_segment`` algorithm when used with unified shared device memory (USM) and number of items equals to 1.
+- Fixed error in ``inclusive_scan_by_segment`` algorithm when used with unified shared device memory (USM).
+- Fixed error in ``reduce_by_segment`` algorithm when used with a DPC++ execution policy and unified shared device memory (USM).
+
+Changes to Existing Features
+----------------------------
+- Improved performance of ``shift_left``, ``shift_right`` algorithms on vector with ``seq``, ``par``, ``par_unseq``, ``unseq``
```

Comment:
the wording should be re-worked... 
But, essentially the change about usage contiguous iterators is not completed in general. It fixes just  less 1% of parallel algorithms.. So, I would removed this input at all. Other words, I would removed lines 26-29

Reply:
If we measured the performance improvement and know it's substantial, it might be worth mentioning even for a couple of algorithms. If we just assume improved performance and do not know how big the improvement is, I agree with Mikhail that it makes sense to first do general improvements for contiguous iterators (because the user value of this small improvement seems negligible).

Reply:
Let's remove the note. 
```suggestion
```
#376 affects shift algorithms used with contiguous iterators and ``seq`` policy only. The performance improvement has not been measured and the user value seems too small for the note to be mentioned.

Reply:
Removed.